rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -517,19 +541,69 @@ rocalResize(
     auto input = static_cast<Image*>(p_input);
     try
     {
-        // For the resize node, user can create an image with a different width and height
+        if((dest_width || dest_height || resize_longer || resize_shorter) == 0)
```

Comment:
Lakshmi asked us to add braces for single if condition too ryt?
Did you check with google guide lines?

Reply:
Yes I checked that they had mentioned single if conditions do not require curly braces
if there is no else condition for that